### PR TITLE
[PR] Readme v2

### LINKED ISSUE #200: [DOC] ReadMe V2
**Que manque-t-il dans la documentation ?**
readme V2 : 
- Easy tutorial to setup local dev with docker and mysql
- More detailled info about files

### PR DESCRIPTION
**Pull request by lduf**
### Description
Rewrite the N4P documentation

## Issues fixed
List of related issues that are fixed by this PR :

- Closes #200 

### Checklist
- [x] Compiling
- [x] Code factoring (comments, indent...)
- [x] Documentation (net4p Doc, Readme...)

### Feature Checklist
- [ ] Cytoscape
- [ ] Person/Index
- [ ] Person/Filter
- [ ] Person/Create
- [ ] Person/Edit
- [ ] Person/Delete
- [ ] Relation/Index
- [ ] Relation/Filter
- [ ] Relation/Create
- [ ] Relation/Edit
- [ ] Relation/Delete
- [ ] Duplicates/Index
- [ ] Duplicates/Filter
- [ ] Duplicates/Create
- [ ] Duplicates/Edit
- [ ] ManageList/Delete
- [ ] ManageList/Create
- [ ] ManageList/Edit
- [ ] ManageField/Index
- [ ] ManageField/Create
- [ ] ManageField/Edit
- [ ] ManageField/Delete
- [ ] ApiList
- [ ] Crews
- [ ] Users

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,353 +1,165 @@
...
-```
-npm update
-composer update
-chmod -R 775 storage
-php artisan migrate --seed
-```
-
-# Connection à l'API
+You can go to [https://portainer.netw4ppl.tech](https://portainer.netw4ppl.tech) to manage the docker server but also
+your machine locally, provided you register your machine.
```
Comment (mathisfchx) [inline README.md:13]: "provided you register your machine" : is it like "providing you the ability to register your machine" ?

File: README.md [hunk 2]
```diff
@@ -1,353 +1,165 @@
...
+**IMPORTANT** :
 
-- Que les deux refugees (personnes) soient déjà présente dans la database. Autrement dit, il faut d'abord effectuer une
-  requete POST add refugee
-- Préciser le `unique_id de chaque personne et la date à laquelle le champ a été créé
-- Préciser une relation appartenant à la liste des relations prédéfinies.
+It is necessary to **add the environment variables** which define the identifiers of the database.
 
-On peut aussi préciser un détail sur la relation
+Your must correctly create and associate the volumes (*you can create theses directories wherever you while, while you
```
Comment (mathisfchx) [inline README.md:46]:
```suggestion
Your must correctly create and associate the volumes (*you can create theses directories wherever you want, as long as you
```

File: README.md (hunk 1 above)
Comment (lduf) [reply README.md:13]: yes

**Review by mathisfchx — APPROVED** [review]
🌍

**Pull request merged**